src/System.Runtime.InteropServices.RuntimeInformation/src/RuntimeInformation.cs

```diff
@@ -19,7 +20,7 @@ public static string FrameworkDescription
         {
             get
             {
-                return FrameworkName;
+                return string.Format("{0} {1}", FrameworkName, typeof(object).GetTypeInfo().Assembly.GetName().Version);
```

Comment:
@weshaggard we can enable file version info using the UNIX impl on .netnative, though it still feels a bit heavy.

That still wouldn't be correct on .net native since the file with object in it would be the app.  Maybe NETNative just transforms the IL to burn in the number.

Burning in a number during the build of this assembly would work some of the time.  It's at least a min version, but if someone upgraded just this package they'd get a newer version.